### PR TITLE
Remove and integrate content from contributing.rst

### PR DESCRIPTION
**Pull request by shamoon**
## Proposed change

This PR updates the documentation to remove `contributing.rst` and integrates the content from that into `extending.rst` as well as links to `CONTRIBUTING.md` as discussed in the issue below.

This is part of #158 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other: documentation

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] ~~If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.~~
- [ ] ~~If applicable, I have checked that all tests pass, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#back-end-development).~~
- [x] I have run all `pre-commit` hooks, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/contributing.html#pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: docs/extending.rst [hunk 1]
```diff
@@ -23,6 +23,31 @@ Apart from that, the folder structure is as follows:
...
+=====================================
+
+To ensure a consistent style and formatting across the project source, the project
+utilizes a Git `pre-commit` hook to preform some formatting and linting before a
+commit is allowed. That way, everyone uses the same style and some common issues
+can be caught early on. See below for installation instructions.
+
+Once installed, hooks will run when you commit. If the formatting isn't quite right
+or a linter catches something, the commit will be rejected. You'll need to look at the
+output and fix the issue. Some hooks, such as the Python formatting tool `black`
```
Comment (qcasey) [inline docs/extending.rst:47]:
```suggestion
output and fix the issue. Some hooks such as the Python formatting tool `black`
```

File: docs/extending.rst [hunk 2]
```diff
@@ -23,6 +23,31 @@ Apart from that, the folder structure is as follows:
...
+it fixed for everyone!
+
+Before contributing please review our `code of conduct`_ and other important
+information in the `contributing guidelines`_.
+
+Code formatting with pre-commit Hooks
+=====================================
+
+To ensure a consistent style and formatting across the project source, the project
+utilizes a Git `pre-commit` hook to preform some formatting and linting before a
```
Comment (qcasey) [inline docs/extending.rst:41]:
```suggestion
utilizes a Git `pre-commit` hook to perform some formatting and linting before a
```

File: docs/extending.rst [hunk 3]
```diff
@@ -121,9 +152,8 @@ Testing and code style:
 Front end development
 =====================
 
-The front end is build using angular. I use the ``Code - OSS`` IDE for development.
-
-In order to get started, you need ``npm``. Install the Angular CLI interface with
+The front end is build using Angular. In order to get started, you need ``npm``.
```
Comment (qcasey) [inline docs/extending.rst:155]:
```suggestion
The front end is built using Angular. In order to get started, you need ``npm``.
```

File: src/paperless/version.py [hunk 4]
```diff
@@ -1 +1 @@
-__version__ = (1, 5, 0)
+__version__ = (1, 6, 0)
```
Comment (qcasey) [inline src/paperless/version.py:1]: Should this change be noted in the [org docs](https://github.com/paperless-ngx/org-docs/blob/main/dev-release-cycle.md)?

**Review by qcasey — COMMENTED** [review]
Found a few small typos, otherwise LGTM. Thanks for this!

File: src/paperless/version.py (hunk 4 above)
Comment (shamoon) [reply src/paperless/version.py:1]: Oh yea I forgot to mention this, my bad. I noticed it a little while ago, and yes I added to docs!

**Comment by shamoon** [comment]
Cool, made changes.

File: src/paperless/version.py (hunk 4 above)
Comment (qcasey) [reply src/paperless/version.py:1]: Awest

**Review by qcasey — APPROVED** [review]

**Pull request merged**